find/templates/find/main/data-glossary.html

```diff
@@ -110,11 +110,6 @@ <h3 id="fund-source-type" class="govuk-heading-s">Fund source type </h3>
                 },
                 "content": {
                     "html": '
-                      <h3 id="geography" class="govuk-heading-s">Geography</h3>
-                      <p class="govuk-body">Usually this depends on the fund and how the data is being collected.  It can include rural, coastal and urban areas as well as postcodes.
-                        Depending on the fund, this data may be collected using <a href="https://ec.europa.eu/eurostat/web/nuts/background">NUTS</a> (Nomenclature of territorial units for statistics) or
-                        <a href="https://www.ons.gov.uk/methodology/geography/ukgeographies/eurostat#toc">ITL 1</a> (International Territorial Levels).
-                      </p>
```

Comment:
Do we definitely want to drop this? We do still collect geographical data (postcodes) for projects 🤔

Reply:
Added it back to the glossary.

Reply:
Thanks, Sam!!!